kadi/commands/commands_v2.py

```diff
@@ -969,31 +1001,96 @@ def is_google_id(scenario):
     return scenario is not None and len(scenario) > 35
 
 
-def update_cmd_events(scenario=None) -> Table:
-    """Update local cmd_events.csv from Google Sheets for ``scenario``.
+def update_cmd_events(
+    scenario=None,
+    event_filter: Callable | list[Callable] | None = None,
+) -> Table:
+    """Update local cmd_events.csv from Google Sheets and read events for ``scenario``.
+
+    For local scenarios, the cmd_events.csv file is read directly from the CSV file in
+    ``paths.CMD_EVENTS_PATH(scenario)``. For the "flight" or None scenario, the
+    cmd_events are read from the Google Sheet and then written to that local file.
+
+    No event filtering is done prior to writing events from a sheet to scenario file.
+
+    After reading the events, the events are filtered based on the "State" column. By
+    default, only events with a "State" of "Predictive" or "Definitive" are included. If
+    the configuration parameter ``conf.include_in_work_command_events`` is True, then
+    events with a "State" of "In-work" are also included.
```

Comment:
Seems like this docstring should mention that they will be filtered based on event_filter as well.